src/agor/tools/agent_handoffs.py

```
@@ -286,57 +501,199 @@ def generate_mandatory_session_end_prompt(
+**Environment**: {environment_info.get('mode', 'unknown')} ({environment_info.get('platform', 'unknown')})
 
 ## 📝 FEEDBACK CONTENT
 
-{feedback_content}
+{feedback_content}"""
+
+    # Add bug-specific sections
+    if feedback_type == "bug" and (reproduction_steps or expected_behavior or actual_behavior):
+        meta_content += f"""
+
+## 🔍 BUG DETAILS"""
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)